tests/test_pdf.py

```diff
@@ -1,8 +1,29 @@
+import re
+from pathlib import Path
+from typing import List
+
 import pytest
 
 
 @pytest.mark.sphinx("latex", testroot="simple")
-def test_simple(tex: str):
-    assert r"\sphinxincludegraphics[]" \
-           r"{drawio-6cf867c6e94665d8489581a35c2d215220d6c152.pdf}" in tex
+def test_simple(tex_images: List[Path]):
+    image, = tex_images
+    assert image.basename() == "drawio-6cf867c6e94665d8489581a35c2d215220d6c152.pdf"
+
+
+@pytest.mark.sphinx("latex", testroot="simple")
+def test_crop(tex_images: List[Path]):
+    image, = tex_images
+    assert get_mediabox(image) == ("88.080002", "46.079998")
+
 
+def get_mediabox(filename):
+    with filename.open("rb") as pdf:
+        for line in pdf.readlines():
+            if line.startswith(b"/MediaBox"):
+                break
+        else:
+            assert False
+    m = re.match(rb"/MediaBox \[0 0 ([\d.]+) ([\d.]+)\]", line)
+    assert m
+    return tuple(g.decode("ascii") for g in m.groups())
```

Comment:
ending newline please

Reply:
> ending newline please

The file ends with an empty line, AFAICS. GitHub doesn't appear to show it, however.

Reply:
Okay.
Took a look at a few other projects and this seems to be everywhere. 😩